scripts/launch-s0-instance.sh

```diff
@@ -393,7 +395,7 @@ create_s0_bootstrap
 echo "Launching \"$instance_name\" (instance type $instance_type) with the latest Elastio s0 AMI ($latest_ami)"
 
 aws_subnet_args=""
-[ -n "$subnet_id" ] && aws_subnet_args="--subnet-id $subnet_id"
+[ -n "${subnet_id-}" ] && aws_subnet_args="--subnet-id $subnet_id"
```

Comment:
probably an issue was here. i could reproduce it having a default vpc and not specifying subnet-id with the modification of the script by adding `set -u`.